assets/css/app.css

```diff
@@ -272,3 +272,15 @@ ul.sidebar .sidebar-list a.active {
   border-left: 3px solid #fff;
   background: #2d3e63;
 }
+
+.sortable {
+  cursor: pointer;
+}
+
+.st-sort-ascent .glyphicon:before {
```

Comment:
Is it possible to get rid of glyphicon and replace the sorting icons by font-awesome icons? Just to stay coherent, we're using font-awesome everywhere except for these sorting icons :)